core/src/test/java/io/cucumber/core/plugin/URLOutputStreamTest.java

```diff
@@ -189,7 +189,8 @@ private Reader openUTF8FileReader(final File file) throws IOException {
                 try {
                     // Every thread should wait command to run
                     countDownLatch.await();
-                    new URLOutputStream(tmp.toURI().toURL());
+                    URLOutputStream urlOutputStream = new URLOutputStream(tmp.toURI().toURL());
+                    urlOutputStream.close();
```

Comment:
Nice catch. But let's use try-with-resources here.

Reply:
I was not sure about Java level you aim for and if I can use t-w-r.

Reply:
Its java 8. It's hidden somewhere in the parent pom.